superset-frontend/src/components/DropdownContainer/index.tsx

```diff
@@ -249,6 +249,14 @@ const DropdownContainer = forwardRef(
       [ref],
     );
 
+    // Closes the popover when scrolling on the document
+    useEffect(() => {
+      document.onscroll = () => setPopoverVisible(false);
```

Comment:
Can we add throttling? Or call it only if popover is open?

Reply:
Or we could even add the scroll listener when popover opens and remove it when popover closes

Reply:
Good suggestion. I changed the code to remove the listener when closing the popover and tested that it does not require throttling.